cluster/manifests/skipper/daemonset.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   namespace: kube-system
   labels:
     application: skipper-ingress
-    version: v0.10.112
+    version: v0.10.121-1-g5f1806d
```

Comment:
Should we cherry-pick it to `.112`, considering that `.121` wasn't really tested?

Reply:
Only filters added which we do not enable by default and one fix for kubernetes in GCP. 
I think it's fine.